### PR TITLE
Add support for SSL certificates and proxy.

### PR DESCRIPTION
**Pull request by pelov**

### CODE REVIEW

**Comment by rockbone** [comment]
@pelov 

I'm sorry I'm late to response.
I'll check fixes and merge.
Thank you.

File: lib/Net/APNS/Simple.pm [hunk 1]
```diff
@@ -63,30 +101,41 @@ sub _client {
...
-        },
+    my @headers = (
+        'apns-topic' => $self->bundle_id,
     );
+
+    for (qw/apns_id apns_priority apns_expiration apns_collapse_id/) {
+        my $v = $self->{$_};
+        next unless defined $v;
+        my $k = $_;
+        $k =~ s/-/_/g;
```
Comment (rockbone) [inline lib/Net/APNS/Simple.pm]: Is it correct?
you mean below?

```suggestion
        $k =~ s/_/-/g;
```

Comment (pelov) [reply lib/Net/APNS/Simple.pm]: Yes, you are correct. Also there was a bug on the next line where it should be `push @headers, $k => $v;`

**Comment by rockbone** [comment]
I merged your branch and made several minor modifications.
Thanks!

**Pull request closed without merging**